src/olympia/search/tests/test_filters.py

```diff
@@ -838,7 +838,7 @@ def test_search_by_author(self):
         filter_ = qs['query']['bool']['filter']
         assert len(filter_) == 1
         should = filter_[0]['bool']['should']
-        assert {'terms': {'listed_authors.id': ['123', '456']}} in should
+        assert {'terms': {'listed_authors.id': [123, 456]}} in should
```

Comment:
I note this change in the ES query but using the integers seems more "correct" anyway?

Reply:
Both should work equally